Tests/ApolloCodegenInternalTestHelpers/MockApolloCodegenConfiguration.swift

```diff
@@ -2,7 +2,7 @@
 
 extension ApolloCodegenConfiguration {
   public static func mock(
-    schemaName: String = "MockSchemaTypes",
+    schemaName: String = "TestSchema",
```

Comment:
This change is because there are two places where the schema name is used; `IR.Schema.name` and `ApolloCodegenConfiguration.schemaName`. In production these can never differ because the IR schema name is initialized from the codegen config schema name. However in test mocks they can be initialized independently and we have been doing that which results in two different namespaces in expected test code. This 'standardizes' the default schema name across both mock objects.

I don't feel strongly about whether this needs to be 'fixed' but it might be worth looking into the places that don't have access to the config and use `IR.Schema.name`; can they be changed to use config instead. For now this is fine though.